### PR TITLE
Gitpod - add extensions + custom image to get quarkus CLI + contributing.md

### PR DESCRIPTION
**Pull request by jeanphi-baconnais**
Hi, 

Using GitPod, I've just changed few small things : 
- add extensions in .gitpod.yml.
- add Dockerfile to get quarkusCLI available in this project
- add information in contributing.md

What do you think about these changes ?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -288,6 +289,11 @@ navigate to _Editor_ -> _Code Style_ -> _Java_ -> _Imports_
 and set _Class count to use import with '\*'_ to `999`.
 Do the same with _Names count to use static import with '\*'_.
 
+### GitPod
+
+You can also use [GitPod](https://gitpod.io) to contribute without install anything on your computer. Click [here](#https://gitpod.io/https://github.com/quarkusio/quarkus/-/tree/main/) to start a workspace.
```
Comment (gsmet) [inline CONTRIBUTING.md:294]: I'm surprised by the `#` at the beginning of the URL?

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+image:
+  file: .gitpod/Dockerfile
+  
+tasks:
+  - init: ./mvnw install -DskipTests=false
```
Comment (gsmet) [inline .gitpod.yml:8]: Should it be `./mvnw -Dquickly` instead? I'm not exactly sure when this task is executed?

File: .gitpod/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    rm -rf $HOME/.sdkman/archives/* && \
+    rm -rf $HOME/.sdkman/tmp/* "
+
+# install jbang 
+RUN bash -c '$HOME/.sdkman/candidates/jbang/current/bin/jbang --version'
+
+RUN bash -c '$HOME/.sdkman/candidates/jbang/current/bin/jbang trust add https://repo1.maven.org/maven2/io/quarkus/quarkus-cli/'
+
+# install quarkus cli
+RUN bash -c '$HOME/.sdkman/candidates/jbang/current/bin/jbang app install --fresh --force quarkus@quarkusio'
```
Comment (gsmet) [inline .gitpod/Dockerfile:20]: Given you are installing `sdkman`, just use it to install Quarkus: `sdk install quarkus`.

**Review by gsmet — CHANGES_REQUESTED** [review]
Not familiar with GitPod but I added a couple of suggestions.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jeanphibaconnais) [reply CONTRIBUTING.md:294]: oh the # isn't at the good location. It's https://gitpod.io/#https://github.com/quarkusio/quarkus/-/tree/main/. I will change now

File: .gitpod.yml (hunk 2 above)
Comment (jeanphibaconnais) [reply .gitpod.yml:8]: This command was already there. I didn't want to change it.

Comment (jeanphibaconnais) [reply .gitpod.yml:8]: but I can change it this command is better.

Comment (gsmet) [reply .gitpod.yml:8]: Already where? This is a new file you are adding :).

**Review by gsmet — CHANGES_REQUESTED** [review]
I spotted a few minor things. Sorry for not spotting them the first time.

Can you adjust and squash your commits?

File: .gitpod.yml (hunk 2 above)
Comment (jeanphibaconnais) [reply .gitpod.yml]: 🤔 I didn't remember.. I changed that.

**Comment by jeanphibaconnais** [comment]
Thanks for your review and help @gsmet  🙏

**Review by gsmet — APPROVED** [review]

**Comment by jeanphibaconnais** [comment]
Hi @gastaldi, I was hesitant to add this button 😅 Tried to add this one under badges in the README.md file but wasn't sure if this was the right place. And in the contributor file, there is so much information so 🤷‍♂️ but I could add this one

File: .gitpod/Dockerfile [hunk 4]
```diff
@@ -0,0 +1,8 @@
+FROM gitpod/workspace-java-11
+
+RUN bash -c ". /home/gitpod/.sdkman/bin/sdkman-init.sh && \
+    sdk install java 11.0.9.j9-adpt && \
```
Comment (maxandersen) [inline .gitpod/Dockerfile:4]: why use j9 here and not adoptium which we use everywhere else ?

Comment (jeanphibaconnais) [reply .gitpod/Dockerfile:4]: oh ! I didn't see that. After reading the project, I didn't find the image to use. Where I can find it please ?

**Pull request merged**